guide.md

```diff
@@ -7459,7 +7459,7 @@ Finally, Open-Source apps should always be preferred because they allow third pa
 <td>Lacks PFS, deniability</td>
 </tr>
 <tr class="odd">
-<td>Signal</td>
+<td>Signal/Molly</td>

```

Comment:
They are mutually exclusive; one cannot be used while the other is being used, and it makes no sense to use both at the same time - one is hardened, the other is not. We should at some point define this later. It's okay as-is, but this will have to be explained in another section about Molly's servers and architecture being hardened. They [Molly devs] also plan to implement their own servers (currently, they are utilizing Signal's servers).

Reply:
It deserves its own section.